sql/core/src/test/scala/org/apache/spark/sql/streaming/TransformWithMapStateTTLSuite.scala

```diff
@@ -308,7 +310,6 @@ class TransformWithMapStateTTLSuite extends TransformWithStateTTLTest {
         AddData(inputStream, MapInputEvent("k1", "", "get_values_in_ttl_state", -1)),
         AdvanceManualClock(1 * 1000),
         CheckNewAnswer(
-          MapOutputEvent("k1", "key3", -1, isTTLValue = true, 123000),
```

Comment:
could we verify this ?

Reply:
Just looked. The test is verifying that the given user-key, `key3`, has its new update timestamp in the secondary index. The old (stale) one should be removed. Thus, 123000 is removed, and 126000 is the new one. This is map state, not list state, so the minimum isn't preserved.